test/integration/feature-flags_test.go

```diff
@@ -0,0 +1,81 @@
+//go:build integration
+
+package integration
+
+import (
+	"testing"
+
+	featureFlagsConstants "github.com/checkmarx/ast-cli/internal/constants/feature-flags"
+	commonParams "github.com/checkmarx/ast-cli/internal/params"
+	"github.com/checkmarx/ast-cli/internal/wrappers"
+	"github.com/spf13/viper"
+	"gotest.tools/assert"
+)
+
+type MockFeatureFlagsWrapper struct {
+	AllFlags     *wrappers.FeatureFlagsResponseModel
+	SpecificFlag *wrappers.FeatureFlagResponseModel
+	Err          error
+}
+
+func TestHandleFeatureFlagsIntegration(t *testing.T) {
```

Comment:
Please change all test names to given when then name (when calling x with x it should do x).. It really hard to u understand what the test should do..

Reply:
Done